Main.cs

```diff
@@ -21,6 +19,8 @@ public class Main : IPlugin, IPluginI18n, IContextMenu
 
         public const string IconPath = "Images/windowwalker.light.png";
 
+        public const int cachedWindowsScore = 100;

```

Comment:
I think this name shouldn't be cachedWindowsScore? because it is not used for cachedwindows, but for all windows?

Reply:
This 100 is from cached window results code though, what do u mean?

Reply:
Oh Sorry I misunderstand the part of code. I thought it was the part of code that used to generate results for empty query.